misc/scripts/upgrade_test.sh

```diff
@@ -19,6 +19,19 @@
     GO="go"
     DEPLOY_TOOLS_SH=../misc/scripts/deploy-tools.sh
 
+    PKG_VERSION=$(git describe --tags `git rev-list --tags --max-count=1` \
+	       ).$(git log --pretty=format:'%h' -n 1)
+    VIBFILE=vmware-esx-vmdkops-$PKG_VERSION.vib
+    VIB_BIN=../build/$VIBFILE
+
+    # variable used to construct vib file and plugin name for customer build
+    PLUGIN_NAME=$DOCKER_HUB_REPO/docker-volume-vsphere
+    GIT_SHA=$(git rev-parse --revs-only --short HEAD)
+    GIT_TAG=$(git describe --tags --abbrev=0 $GIT_SHA)
+    VERSION_TAG=$(echo $GIT_TAG | awk -F. '{printf ("%d.%d", $1, $2+1)}' )
+    EXTRA_TAG=-dev
+    PLUGIN_TAG=$VERSION_TAG$EXTRA_TAG
```

Comment:
No, L22-33 is not necessary  ... required information should be passed from make file to here instead of reinitializing here.